frontend/src/scenes/web-analytics/tiles/WebAnalyticsTile.tsx

```diff
@@ -283,6 +283,16 @@ const BreakdownValueCell: QueryContextColumnComponent = (props) => {
                 )
             }
             break
+        case WebStatsBreakdown.OS:
+            if (typeof value === 'string') {
+                return (
+                    <div className="flex items-center gap-2">
```

Comment:
This `flex items-center gap-2` class is now repeated 3 times, any chance you could make this a component? `PropertyIconWithLabel` maybe?

Reply:
Sounds good :)

Reply:
@rafaeelaudibert what's the direction for those small components, should I just add it to the lib/components since it may be somewhat useful to other places or add it to our web-analytics cozy space somewhere? I am not sure we have system design directives or haven't got to that part of the onboarding handbook so far.

Reply:
Sound good, I am happy to do it but just to make sure, the component here is `PropertyIcon`, not `Filter`, which already have those other stuff there like logics, etc... 

The `PropertyIcon` is this single file with an assigned storybook story.

That is the one you think would make sense to turn into a compound component, right?

Reply:
Gotta get some ☕. Yep, that's the one I meant :)